.github/workflows/php-tests.yml

```diff
@@ -10,26 +10,56 @@ jobs:
       fail-fast:    false
       max-parallel: 10
       matrix:
-        woocommerce: [ 'latest', '5.7.1', '5.6.1' ]
-        wordpress:   [ 'latest', '5.7', '5.6' ]
-        php:         [ '8.0', '7.4', '7.0' ]
-    name: Stable [PHP=${{ matrix.php }}, WP=${{ matrix.wordpress }}, WC=${{ matrix.woocommerce }}]
+        woocommerce_L: [ 'L', 'L-1', 'L-2' ]
```

Comment:
I don't really like this name, but couldn't think of anything meaningful 🤷🏼

Reply:
Maybe it could be `woocommerce_policy`? Or `woocommerce_version_offsets`? Or `woocommerce_support`?

I think `X_support_policy` or `X_policy` might be good? I definitely don't think we should leave it at `X_L`, that doesn't really tell us anything 😛